src/builtins/core/date.rs

```diff
@@ -180,6 +180,62 @@ impl PartialDate {
 }
 
 /// The native Rust implementation of `Temporal.PlainDate`.
+///
+/// Represents a calendar date like "March 15th, 2024" without any time or timezone
+/// information. Useful for dates where the specific time of day doesn't matter,
+/// such as deadlines, birth dates, or historical events.
+///
+/// Uses the ISO 8601 calendar (standard Gregorian calendar) by default, with
+/// support for other calendar systems when needed.
+///
+/// ## Examples
+///
+/// ```rust
+/// use temporal_rs::{PlainDate, Calendar};
+///
+/// // Create a date using the ISO calendar
+/// let christmas = PlainDate::try_new_iso(2024, 12, 25).unwrap();
+///
+/// // Explicit calendar specification
+/// let date = PlainDate::try_new(2024, 12, 25, Calendar::default()).unwrap();
+/// ```
+///
+/// Date arithmetic operations:
+///
+/// ```rust
+/// use temporal_rs::{PlainDate, Duration};
+/// use std::str::FromStr;
+///
+/// let start = PlainDate::try_new_iso(2024, 1, 15).unwrap();
+///
+/// // Add one month
+/// let later = start.add(&Duration::from_str("P1M").unwrap(), None).unwrap();
+/// // Results in February 15th, 2024
```

Comment:
```suggestion
/// // Results in 2024-02-15
```

nit: i18n sensitive language.

It may be worthwhile working this into the preceding comment too, but that's not necessarily needed. It would be fine left in the current place too.